packages/cli/src/errors/max-stalled-count.error.ts

```diff
@@ -1,9 +1,9 @@
-import { ApplicationError } from 'n8n-workflow';
+import { UserError } from 'n8n-workflow';
 
 /**
  * @docs https://docs.bullmq.io/guide/workers/stalled-jobs
  */
-export class MaxStalledCountError extends ApplicationError {
+export class MaxStalledCountError extends UserError {
```

Comment:
This is more like a transient error, so we should extend `OperationalError` instead